docs/testing/releases/600.md

```diff
@@ -0,0 +1,192 @@
+- Change the styles in the post/page editor and verify they have priority over the styles from the Site editor.
+- Repeat the process for the Reviews by Category and Reviews by Product blocks.
+![imatge](https://user-images.githubusercontent.com/3616980/121201895-2a386580-c875-11eb-928b-34ca0a1be531.png).
+
+
+### Update All Reviews block so it honors 'ratings enabled' and 'show avatars' preferences. ([4764](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4764))
+
+
+1. In wp-admin, go to _Settings_ > _Discussion_ > _Avatars_ and uncheck _Show Avatars_.
+2. Add an All Reviews block and verify avatars are not shown and instead there is a warning in the sidebar:
+
+<img src="https://user-images.githubusercontent.com/3616980/133635801-013aacf7-e6e3-48ac-978e-b36f81284c4a.png" width="298" alt="" />
+
+3. In wp-admin, go to _WooCommerce_ > _Settings_ and uncheck _Enable star rating on reviews_.
+4. In the All Reviews block, verify ratings and the _Sort by_ select aren't displayed.
+
+
+### Fix infinite recursion when removing an attribute filter from the Active filters block. ([4816](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4816))
+
+- Insert All Products, Filter Products by Attribute and Active filter blocks into a page.
+- Publish the page.
+- Go to Frontend on same page which created in above step.
+- Apply filter for any attribute.
+- Verify you can remove attribute filters either using the cross icon next to it or pressing Clear All.
+
+
+### Products by Category: Moved renderEmptyResponsePlaceholder to separate method to prevent unnecessary rerender. ([4751](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4751))
+
+- Go to a new or existing page and add the "Products by Category" block
+- Select a category with products in it.
+- Select and unselect the block.
+- The block should no longer go into loading state and rerender products again.
+
+
+### Fix calculation of number of reviews in the Reviews by Category block. ([4729](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4729))
+
+- Add some reviews to some of your products.
+- Create a page and add the Reviews by Category block.
+- Verify the counter shows the correct number of reviews. Take special attention to categories with more than one review and categories with subcategories.
+
+#### Screenshots
+
+| Before | After |
+| --- | --- |
+| ![Before screenshot](https://user-images.githubusercontent.com/3616980/132999205-923856f1-47f7-4a34-b3e4-c823ec9c5623.png) | ![After screenshot](https://user-images.githubusercontent.com/3616980/132999179-691761f0-1396-459d-a3f8-f79261bcd9b4.png) |
+
+### Removed `wp-blocks` dependency from several frontend scripts. ([4767](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/4767))
+
+- Open `build/cart-frontend.asset.php`, `build/cart-frontend.asset.php` for example.
+- There should be no `wp-blocks` there.
```

Comment:
My fault with these testing steps, they aren't written from an end user point of view. :man_facepalming: 

Can we change them with something like:
```suggestion
- Open a page with the Cart block.
- In the browser devtools, open the Network tab.
- Search for a JS script with this part in the name `blocks/index.min.js`.
- Verify it isn't there meaning it was loaded.
```

Reply:
It's okay, you write excellent testing steps every time.